src/scripts/validation.js

```diff
@@ -95,7 +94,7 @@ export const enableValidation = (config) => {
 };
 
 //activates form validation
-enableValidation(settings);
+//enableValidation(settings);
 
 // close modal with Esc
 document.addEventListener("keydown", (evt) => {
```

Comment:
* When I close modals, there are errors https://snipboard.io/myEhu7.jpg


Please, delete the `Escape` and `overlay` handling from the `validation.js` file, because everything is correctly done inside `openModal` in `index.js`